Dockerfile

```diff
@@ -0,0 +1,57 @@
+# How to run instructions:
+# 1. Generate ssh command: ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+#    - Save the key in local repo where Dockerfile is placed as id_rsa
+#    - Add the public key to the GitHub account
+# 2. Build docker image: docker build -t ubuntu-ora-dev .
```

Comment:
"ubuntu-opml-dev" would be a better name for the docker image

Reply:
updated image name @0x1cc